packages/instantsearch.js/src/components/RefinementList/RefinementList.tsx

```diff
@@ -270,6 +274,20 @@ class RefinementList<TTemplates extends Templates> extends Component<
     }
   }
 
+  /**
+   * This sets focus on the last refined input element after a render
+   * because Preact does not perform it automatically.
+   * @see https://github.com/preactjs/preact/issues/3242
+   */
+  public componentDidUpdate() {
```

Comment:
too bad to have this fix in the non-unified components 🤔

Reply:
Yes, we should brace for a few headaches when we'll migrate all of this into the unified components package 😰.